hack/quickstart/init-master.sh

```diff
@@ -56,29 +56,26 @@ function init_master_node() {
     fi
 
     # Render cluster assets
-    /home/core/bootkube render --asset-dir=/home/core/assets ${etcd_render_flags} \
+    /home/${REMOTE_USER}/bootkube render --asset-dir=/home/${REMOTE_USER}/assets ${etcd_render_flags} \
       --api-servers=https://${COREOS_PUBLIC_IPV4}:443,https://${COREOS_PRIVATE_IPV4}:443
 
     # Move the local kubeconfig into expected location
-    chown -R core:core /home/core/assets
+    chown -R ${REMOTE_USER}:${REMOTE_USER} /home/${REMOTE_USER}/assets
     mkdir -p /etc/kubernetes
-    cp /home/core/assets/auth/kubeconfig /etc/kubernetes/
-    cp /home/core/assets/tls/ca.crt /etc/kubernetes/ca.crt
+    cp /home/${REMOTE_USER}/assets/auth/kubeconfig /etc/kubernetes/
+    cp /home/${REMOTE_USER}/assets/tls/ca.crt /etc/kubernetes/ca.crt
 
     # Start etcd.
     if [ "$SELF_HOST_ETCD" = false ] ; then
         configure_etcd
         systemctl enable etcd-member; sudo systemctl start etcd-member
     fi
 
-    # Set cloud provider
```

Comment:
It looks like some lines here got removed unrelated to this PR. I think basically a few changes from https://github.com/kubernetes-incubator/bootkube/commit/bc9154ea8c403425f9a69e0ea8385f1bda40e3f1 got reverted. It added cloud provider support to the quickstart scripts.